packages/edit-post/src/components/header/style.scss

```diff
@@ -198,14 +198,24 @@
 	.edit-post-header__toolbar .block-editor-block-mover {
 		border-left: none;
 
+		// Modified group borders
```

Comment:
This comment doesn't seem to be related to the actual pseudo-element style, so how about deleting this comment?

Reply:
This pseudo element is actually the modified border of the toolbar group when the 'Top toolbar' option is enabled:

![Screenshot 2024-01-11 at 14 58 11](https://github.com/WordPress/gutenberg/assets/1682452/ed8ff564-4532-491a-90d0-93173903add9)